projects/angular/src/modal/_modal.clarity.scss

```diff
@@ -168,11 +218,22 @@
   .modal-footer {
     display: flex;
     justify-content: flex-end;
-    padding: variables.$clr_baselineRem_1 0 0 0;
+    @include mixins.css-var(
+      padding,
+      clr-modal-footer-padding,
+      variables.$clr_baselineRem_1 0 0 0,
+      variables.$clr-use-custom-properties
+    );
+    @include mixins.css-var(
+      gap,
+      clr-modal-footer-gap,
+      variables.$clr_baselineRem_0_5,
+      variables.$clr-use-custom-properties
+    );
 
     .btn {
       //switched right margin to left because footer is right aligned
-      margin: 0 0 0 variables.$clr_baselineRem_0_5;
+      margin: 0;
```

Comment:
If I understood this correctly you are removing between button margins and replacing them with gaps. Right?

Reply:
Yeah, it's pretty much the same.